packages/edit-site/src/components/sidebar/spacing-panel.js

```diff
@@ -65,29 +72,54 @@ function useCustomUnits( { units, contextName } ) {
 	return usedUnits.length === 0 ? false : usedUnits;
 }
 
-export default function SpacingPanel( {
-	context: { name },
-	getStyle,
-	setStyle,
-} ) {
+function useCustomSides( blockName, feature ) {
```

Comment:
It looks like this function is exactly the same than `useCustomSides` in the hook, so I wonder if we can consolidate. Fine to do it later as well.

Reply:
Thanks, that [duplication has been removed](https://github.com/WordPress/gutenberg/pull/30607/commits/5d1b856b72a59558b67a1761e4fda75cad3ae4cb). I've exported `useCustomSides` and used this in `spacing-panel.js`